tests/bpf2c_tests/expected/atomic_instruction_fetch_add_dll.c

```diff
@@ -178,7 +178,7 @@ static void
 _get_version(_Out_ bpf2c_version_t* version)
 {
     version->major = 0;
-    version->minor = 6;
+    version->minor = 7;
```

Comment:
Could you `ebpf_version.h` not be refactored to have separate definitions for major, minor and patch version, and have `EBPF_VERSION` be defined as the joining of that? Then all these tests simply include the header and use those definitions?

Reply:
Should be outside the scope of this PR in my view.

Reply:
I am merging but these are good suggestions to do in a separate PR.